apps/nuxt3-ssr/components/global/Copy.vue

```diff
@@ -0,0 +1,9 @@
+<!-- eslint-disable vue/multi-word-component-names -->
```

Comment:
I would have expected the SVG to be apart. And is this generated or manually created/